docs/index.md

```diff
@@ -77,11 +77,17 @@ Currently the most common approach to managing external dependencies is using
 
 ## WebAssembly
 
-To build a `rust_binary` for wasm32-unknown-unknown add the `--platforms=@io_bazel_rules_rust//rust/platform:wasm` flag.
+To build a `rust_binary` for `wasm32-unknown-unknown` target add the `--platforms=@io_bazel_rules_rust//rust/platform:wasm32-unknown` flag.
 
 ```command
-bazel build @examples//hello_world_wasm --platforms=@io_bazel_rules_rust//rust/platform:wasm
+bazel build @examples//hello_world_wasm --platforms=@io_bazel_rules_rust//rust/platform:wasm32-unknown
```

Comment:
I'm not sure if the new name should be `wasm32-unknown` or `wasm32-unknown-unknown` to match Rust's triple.

Reply:
I'd prefer it to match the triple

Reply:
Actually, this doesn't work, since then `@io_bazel_rules_rust//rust/platform:wasm32-unknown-unknown` **platform** collides with `@io_bazel_rules_rust//rust/platform:wasm32-unknown-unknown` **constraint**.

This is actually quite problematic because there is no good way to name `wasm32-wasi` without creating collisions. We need to have `wasm32-wasi` **constraint**, and we need to have `wasi` **os**, but we also want to have **platform** that's either `wasm32-wasi` or simply `wasi`, but we cannot have either right now.

I think triples need to stay where they are, since a lot of external projects depends on them, but since **cpu** and **os** are mostly internal, we could move them elsewhere (e.g. `//rust/platform/cpu:wasm32`, `//rust/platform/os:unknown` and `//rust/platform/os:wasi`).

What do you think? Any preferences?

Reply:
OK, All triple constraints are left in place, I moved the individual constraints to `//rust/platform/cpu:wasm32`, `//rust/platform/os:unknown` and `//rust/platform/os:wasi`, and renamed the new platform to `wasi`.

I think it looks good now, and I cross-tested it with `cargo-raze` and my own project that relies on those.

Reply:
This looks good to me. Is there a paired change needed in `cargo-raze`?